### PR TITLE
Bluetooth Device: Issue ZLP on ACL IN ep when transfer is multiple of endpoint max packet size

### PR DESCRIPTION
**Pull request by donatieng**
The BTH device class implementation doesn't work on Linux hosts for transfers which size are a multiple of the ACL IN endpoint maximum packet size. This PR fixes this issue by issuing a zero-length packet on that endpoint in these cases, similarly to the CDC device class implementation.

### CODE REVIEW

**Comment by donatieng** [comment]
I've rebased the PR and fixed clang-format'ted changes

File: src/class/bth/bth_device.c [hunk 1]
```diff
@@ -127,11 +131,25 @@ uint16_t btd_open(uint8_t rhport, tusb_desc_interface_t const *itf_desc, uint16_
...
+  for (size_t p = 0; p < 2; p++) {
+    if (tu_edpt_dir(desc_ep_acl_in->bEndpointAddress) == TUSB_DIR_IN) {
+      _btd_itf.ep_acl_in_pkt_sz = tu_edpt_packet_size(desc_ep_acl_in);
+      break;
+    }
+    desc_ep_acl_in = (tusb_desc_endpoint_t const *)tu_desc_next(desc_ep_acl_in);
+  }
 
-  itf_desc = (tusb_desc_interface_t const *)tu_desc_next(tu_desc_next(tu_desc_next(desc_ep)));
+  itf_desc = (tusb_desc_interface_t const *)tu_desc_next(tu_desc_next(desc_ep));
```
Comment (HiFiPhile) [inline src/class/bth/bth_device.c:152]: Thank you for the PR, could you elaborate this desc change ?

Comment (donatieng) [reply src/class/bth/bth_device.c:152]: No problem. 

The `tu_desc_next()` iteration that used to be made at that line now happens at line 134. I'm getting a reference to first endpoint descriptor here:
https://github.com/hathach/tinyusb/pull/2783/files#diff-5b2b758cfac5035ee55f2e5189f8c0488362d0c9db1332ba58d064823b3cab9cR134

This allows to recover the max packet size for the IN endpoint here:
https://github.com/hathach/tinyusb/pull/2783/files#diff-5b2b758cfac5035ee55f2e5189f8c0488362d0c9db1332ba58d064823b3cab9cR142

**Review by HiFiPhile — APPROVED** [review]

File: src/class/bth/bth_device.c (hunk 1 above)
Comment (HiFiPhile) [reply src/class/bth/bth_device.c:152]: Thank you, I missed the change in L143.

**Comment by donatieng** [comment]
Thanks @HiFiPhile !

**Pull request merged**